package.json

```diff
@@ -53,61 +53,99 @@
   ],
   "dependencies": {
     "@basicer/geoip-lite": "^1.1.6",
+    "@vimeo/player": "^2.2.0",
     "JQDeferred": "~2.1.0",
-    "ace-builds": "https://github.com/ajaxorg/ace-builds/archive/3fb55e8e374ab02ce47c1ae55ffb60a1835f3055.tar.gz",
+    "ace-builds": "ajaxorg/ace-builds#3fb55e8e374ab02ce47c1ae55ffb60a1835f3055",
     "aether": "~0.5.21",
-    "async": "0.2.x",
+    "algoliasearch": "^3.13.1",
+    "async": "2.5.0",
     "aws-sdk": "~2.0.0",
-    "backbone": "^1.3.3",
+    "babel-loader": "^7.1.2",
+    "backbone": "1.1.0",
     "basic-auth": "^1.0.4",
     "bayesian-battle": "0.0.7",
     "bluebird": "^3.2.1",
     "body-parser": "^1.15.2",
+    "bootstrap": "^3.3.7",
+    "bootstrap-sass": "3.3.7",
```

Comment:
Check  `bootstrap`'s changelog to see if it might have any breaking changes between `~3.2.0` and `^3.3.7`